pages/common/git-verify-commit.md

```diff
@@ -0,0 +1,17 @@
+# git verify-commit
+
+> Check for GPG verification of commits.
+> If GPG verification isn't present, nothing will be returned, even if extra options are chosen.
+> More information: <https://git-scm.com/docs/git-verify-commit>.
+
+- Check git commits for a GPG signature:
```

Comment:
No need to specified that they're _Git_ commits.  Also, it's properly spelled with a capital 'G' i.e. "Git".
```suggestion
- Check commits for a GPG signature:
```